morpheus/_lib/include/morpheus/utilities/stage_util.hpp

```diff
@@ -20,34 +20,44 @@
 #include <algorithm>
 
 namespace morpheus {
-    /****** Component public free function implementations************/
-    /**
-     * TODO(Documentation)
-     */
-    template<typename FuncT, typename SeqT>
-    auto foreach_map(const SeqT &seq, FuncT func) {
-        using value_t = typename SeqT::const_reference;
-        using return_t = decltype(func(std::declval<value_t>()));
+/****** Component public free function implementations************/
 
-        std::vector<return_t> result{};
+/**
+ * @addtogroup utilities
+ * @{
+ * @file
+ */
+
+/**
+ * TODO(Documentation)
+ */
+template <typename FuncT, typename SeqT>
+auto foreach_map(const SeqT &seq, FuncT func)
+{
+    using value_t  = typename SeqT::const_reference;
+    using return_t = decltype(func(std::declval<value_t>()));
+
+    std::vector<return_t> result{};
 
-        std::transform(seq.cbegin(), seq.cend(), std::back_inserter(result), func);
+    std::transform(seq.cbegin(), seq.cend(), std::back_inserter(result), func);
 
-        return result;
-    }
+    return result;
+}
 
-    /**
-     * TODO(Documentation)
-     */
-    template<typename FuncT, typename SeqT>
-    auto foreach_map2(const SeqT &seq, FuncT func) {
-        using value_t = typename SeqT::const_reference;
-        using return_t = decltype(func(std::declval<value_t>()));
+/**
+ * TODO(Documentation)
+ */
+template <typename FuncT, typename SeqT>
+auto foreach_map2(const SeqT &seq, FuncT func)
```

Comment:
I'm actually not clear as to how map2 is different from map1.

Reply:
They both look identical. Not sure why there is two functions. However, `foreach_map2` has not been used anywhere.

Reply:
removed `foreach_map2` function